app/logger/index.js

```diff
@@ -49,4 +44,14 @@ const logger = new winston.Logger({
   ]
 });
 
+if (!config.isTesting) {
+  logger.add(
+    new winston.transports.Console({
```

Comment:
@mishuagopian I could not make this work. I have this error:
```
/home/gescandarini/Wolox/test/express-js-bootstrap/node_modules/winston/lib/winston/logger.js:481
  var instance = created ? transport : (new (transport)(options));
                                        ^

TypeError: transport is not a constructor
    at exports.Logger.Logger.add (/home/gescandarini/Wolox/test/express-js-bootstrap/node_modules/winston/lib/winston/logger.js:481:41)
    at Object.<anonymous> (/home/gescandarini/Wolox/test/express-js-bootstrap/app/logger/index.js:48:10)
```

The rest is ok.